CBinding/Project/CProject.cs

```diff
@@ -45,29 +45,31 @@
 
 namespace CBinding
 {
-	public enum Language {
+	public enum Language
```

Comment:
There's a LOT of formatting changes which make this difficult to read.

When committing, please use a tool like 'git-gui' and stage and commit only the files and lines that you want to include in the commit. You can then revert accidental changes like these without committing them.